src/gui/rss/rssparser.cpp

```diff
@@ -257,7 +257,13 @@ void RssParser::parseRssArticle(QXmlStreamReader& xml, const QString& feedUrl)
         article["title"] = xml.readElementText().trimmed();
       else if (xml.name() == "enclosure") {
         if (xml.attributes().value("type") == "application/x-bittorrent")
```

Comment:
Hmmm I did not know that, and don't have a example as well.
The other effect of this change is that it assigns the enclosure to the "news_link" first, and only assigns back to the "torrent_url" if there are no magnet links. Giving preference the magnet links.
Of course now that I know that "application/x-bittorrent" can't be magnet links, there are other approaches I could go with and still keep the preference on magnet links.